app.js

```diff
@@ -43,7 +44,8 @@ function clientLoaded (client) {
  * @param {Object} err - error from application.
  */
 function errHandler(err) {
-  if(err.name === 'EarlyHangup' || err.name === 'HangupFailure') {
+  if(err.name === 'EarlyHangup' || err.name === 'HangupFailure' ||
```

Comment:
That is already a bit unwieldy and will only grow, I would move that check to a function. fatalError() perhaps?